### ISSUE TITLE
About the annotation of the datasets

### ISSUE THREAD

**Issue by SwanKnightZJP** [M3DV/RibFrac-Challenge#6]
Hi, dear organizer.

During verifying the performance of our method,  we meet some problems with annotations in Training set and Validation Set.
It mainly includes the following two aspects:
1. It seems that some fracture areas in the Validation Set are not annotated. As shown in the figure, there are two abnormal cortical bones, but the annotated area does not include these two places.
![image](https://user-images.githubusercontent.com/37257181/91157299-b7039f80-e6f7-11ea-986f-f9bbd11cf58a.png)
![image](https://user-images.githubusercontent.com/37257181/91157320-be2aad80-e6f7-11ea-8088-38ce66f1b9b6.png)
2. Part of the fracture occurred on multiple ribs, but only have one label. Such as the RibFrac114 in Training set and RibFrac445 in Validation Set. This may be for the convenience of annotating. However, annotations like this with gaps in the middle may have problems while under evaluation. (see line 246 in evaluation.py for details) When the centroid(x, y, z) is located in these gaps, whether it exists possible for the original labels to be classified as the 'Background'?
![image](https://user-images.githubusercontent.com/37257181/91158702-9fc5b180-e6f9-11ea-897a-d72c43d10821.png)
![image](https://user-images.githubusercontent.com/37257181/91158715-a48a6580-e6f9-11ea-950a-2b65f4a7ef18.png)

**Comment by duducheng** [comment]
Hi @SwanKnightZJP ,

Thank you very much for your feedback!

For both issue 1 and 2, I am sorry that the mentioned issues are possible, even though we have tried our best to improve the annotation quality. It may occurs in the human annotation procedure or the pre-processing step. 

We have paid more attention on the test set (than the training and validation), so that the risk of these issues will be reduced.  It is still human annotations anyway, I can not guarentee the 100% accuracy of the annotations though.

We are re-checking the annotations with the hope of a updated version (RibFrac v2), but it is not possible to make it done for the RibFrac Challenge this year. Stay tuned!

Please note that it is also fair the participants this year, since every participants use the same dataset anyhow.  Thanks for understanding. 

Jiancheng Yang
On behalf of MICCAI 2020 RibFrac Challenge Organizers

**Comment by SwanKnightZJP** [comment]
Thank you very much for your reply, and thanks to the RibFrac project team for publishing such an excellent dataset.
Looking forward to the updated version. 
 (╹◡╹)ノ♡
